CRM/Core/Permission.php

```diff
@@ -1367,6 +1367,14 @@ public static function getEntityActionPermissions() {
         'edit contributions',
       ),
     );
+    
+    // Payment Processor Permissions    
+    $permissions['payment_processor'] = array(
+      'get' => array(
+        array('access CiviEvent', 'edit all events'),
```

Comment:
why two arrays?

Reply:
I'm trying to grant ("access CiviEvent" AND "edit all events") OR "administer CiviCRM" as commented [here](https://issues.civicrm.org/jira/browse/CRM-20032?focusedCommentId=100914&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-100914) and I understand this is the way as documented [here](https://github.com/francescbassas/civicrm-core/blob/97549a974fbf1b99875bd8ae65bfab394613a802/CRM/Core/Permission.php#L922)

Reply:
Thanks for enlightening me. Looks good.